build-tests/api-documenter-test/src/DocClass1.ts

```diff
@@ -204,3 +204,25 @@ export class DocClass1 extends DocBaseClass implements IDocInterface1, IDocInter
     return x + y;
   }
 }
+
+/**
+ * Interface without inline tag to test custom TOC
+ * @public
+ */
+export interface IDocInterface5 {
+  /**
+   * string property
+   */
+  string: string;
```

Comment:
>string [](start = 2, length = 6)

Might be better to use member names that aren't also type names.

Reply:
fixed!